### PR TITLE
Add CREDITS.md

### PR DESCRIPTION
**Pull request by duckinator**
Alternative to #64.

### CODE REVIEW

**Comment by duckinator** [comment]
@nbraud I like your idea in #64, but:

1. I think I'd like something a simpler in terms of formatting.
2. I'm considering including software/libraries/services we rely on.

This is what it looks like: https://github.com/duckinator/emanate/blob/c8e2fde/CREDITS.md

What're your thoughts?

File: CREDITS.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- [Ellen Marie Dash (@duckinator)](https://github.com/duckinator)
+- [nicoo (@nbraud)](https://github.com/nbraud)
+
+
+# Emanate Contributors
+
+These are folks who aren't maintainers, but have contributed in some way. :)
+
+- [Samuel Lewis (@ritlew)](https://github.com/ritlew)
+
```
Comment (nbraud) [inline CREDITS.md]:
```suggestion
- [@kayila](https://github.com/kayila)

```

She caused some interesting/useful discussion in #54 (which I should get off my butt and act on).

File: CREDITS.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Emanate Maintainers
```
Comment (nbraud) [inline CREDITS.md]: I think we can shorten it:
```suggestion
# Maintainers
```

File: CREDITS.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+- [Ellen Marie Dash (@duckinator)](https://github.com/duckinator)
+- [nicoo (@nbraud)](https://github.com/nbraud)
+
+
+# Emanate Contributors
```
Comment (nbraud) [inline CREDITS.md]: Same
```suggestion
# Contributors
```

File: CREDITS.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+
+# Software and Services We Rely On ♥
+
+All of the following help make Emanate as awesome as it is:
+
+Software and libraries:
+
+- [Python](https://python.org)
+- [flake8](https://gitlab.com/pycqa/flake8)
```
Comment (nbraud) [inline CREDITS.md]:
```suggestion
- [flake8](https://gitlab.com/pycqa/flake8) and its [bugbear](https://github.com/PyCQA/flake8-bugbear), and [commas](https://github.com/PyCQA/flake8-commas/) plugins;
- [pydocstyle](https://github.com/PyCQA/pydocstyle/), though the [docstrings](https://gitlab.com/pycqa/flake8-docstrings) plugin for flake8;
```

Comment (nbraud) [reply CREDITS.md]: Perhaps don't use inline links here, it's getting kinda unwieldy?

File: CREDITS.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+- [pip](https://pip.pypa.io/)
+- [pylint](https://www.pylint.org/)
```
Comment (nbraud) [inline CREDITS.md]:
```suggestion
- [pylint](https://www.pylint.org/) and the [pytest-pylint](https://github.com/carsongee/pytest-pylint) integration;
```

File: CREDITS.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+- [pytest](https://pytest.org/)
+- [setuptools](https://github.com/pypa/setuptools)
```
Comment (nbraud) [inline CREDITS.md]:
```suggestion
- [setuptools](https://github.com/pypa/setuptools) and [wheel](https://github.com/pypa/wheel);
```

File: CREDITS.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+All of the following help make Emanate as awesome as it is:
+
+Software and libraries:
+
+- [Python](https://python.org)
+- [flake8](https://gitlab.com/pycqa/flake8)
+  - flake8 [bugbear](https://github.com/PyCQA/flake8-bugbear) plugin
+  - flake8 [commas](https://github.com/PyCQA/flake8-commas/) plugin
+- [pydocstyle](https://github.com/PyCQA/pydocstyle/) through the [docstrings](https://gitlab.com/pycqa/flake8-docstrings) plugin for flake8
```
Comment (nbraud) [inline CREDITS.md]:
```suggestion
  - [docstrings](https://gitlab.com/pycqa/flake8-docstrings) plugin to integrate with [pydocstyle](https://github.com/PyCQA/pydocstyle/)
```

**Pull request merged**